community/cloud-foundation/templates/pubsub/README.md

```diff
@@ -1,11 +1,11 @@
 # Pubsub
 
-Templated Pub/Sub deployment
+This template creates a Pub/Sub (publish-subscribe) service.
 
 ## Prerequisites
 
 - Install [gcloud](https://cloud.google.com/sdk)
-- Create a [GCP project, setup billing, enable requisite APIs](../project/README.md)
+- Create a [GCP project, set up billing, enable requisite APIs](../project/README.md)

```

Comment:
Hey @fawix, if I'm not mistaken, you don't need these permissions to run the template. Topic/subscription's IAM policy is set during resource creation (not as a separate step) and editor-level role, which DM's service account has by default, is enough. However, your account does need those permissions to view/edit pubsub's IAM and those permissions aren't something given by default.
If you want, we can add something like ` - Grant [pubsub.admin](https://link) role to an account that will view/edit topic/subscription's IAM policy after its creation.`. Please let me know if we should go with that.

Reply:
@sourced-pavel that is interesting, because I had to add those to my DM service account to successfully run the template (not to user account).  
Are you able to successfully run this template without those on the DM service account? My DM service account already had editor role btw.

Reply:
@fawix , I was wrong. We did have pubsub.admin role granted to our DM service account, but it was applied at organization level, whereas my permissions start at project level. I'll add a note about `pubsub.admin` to the README